db/schema.rb

```diff
@@ -10,11 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_06_17_142340) do
+ActiveRecord::Schema.define(version: 2019_06_26_155821) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "access_limits", force: :cascade do |t|
+    t.datetime "created_at"
+    t.bigint "created_by_id"
+    t.bigint "edition_id", null: false
```

Comment:
I'm somewhat surprised this works as well, noticing you mentioned the expected race condition in your commit. However, should it actually b consistent with statuses, which don't have a not null requirement?

Reply:
This works because Edition has an optional association with an Access Limit. Which allows us to have a required one on the AccessLimit. 

This isn't the case with Status as an Edition requires it.